recipes/_carbon_config.rb

```diff
@@ -17,6 +17,14 @@
 # limitations under the License.
 #
 
+directory "conf dir" do
+  path "#{node['graphite']['base_dir']}/conf"
+  owner node['graphite']['user']
+  group node['graphite']['user']
```

Comment:
That was definitely a mistake on my part but there are better ways to point it out than sarcastically insulting the folks who are trying to make things better.